test/server/cards/03_TWI/units/PloKoonKohtoyah.spec.ts

```diff
@@ -0,0 +1,44 @@
+describe('Plo Koon Kohtoyah', function () {
+    integration(function (contextRef) {
+        describe('Plo Koon Kohtoyah\'s ability', function () {
+            it('should ambush attack for 6', function () {
+                contextRef.setupTest({
+                    phase: 'action',
+                    player1: {
+                        hand: ['plo-koon#kohtoyah'],
+                        groundArena: ['obiwan-kenobi#following-fate', 'luke-skywalker#jedi-knight'],
+                    },
+                    player2: {
+                        groundArena: ['wampa'],
+                    }
+                });
+
+                const { context } = contextRef;
+
+                context.player1.clickCard(context.ploKoon);
+                context.player1.clickPrompt('Ambush');
+                expect(context.wampa).toBeInZone('discard');
+                expect(context.ploKoon.damage).toBe(4);
+                expect(context.ploKoon.exhausted).toBeTrue();
+            });
+
+            it('should defend back for 3 damage', function () {
```

Comment:
can you merge both test in one ?

Reply:
Not sure how to achieve that. 
How can I reset and create a new test setup inside a unique test?

Reply:
you don't need to reset your test, you can find a setup which fit your need, for exemple : plo koon ambush into consular security force, expected damage on csf is 6, opponent's wampa attack into plo koon, expected damage  on wampa is 3